### DISCUSSION TITLE
Date/time stamps on the list of discussions

### DISCUSSION THREAD

**Discussion by GrampaWildWilly** [community/community#102968] [Discussions]
### Select Topic Area

General

### Body

When you go into a Discussions list & you get the list of discussion titles, there is a line in each entry showing the date of the original post.  It would be really great if it showed the date/time of the most recent comment within the discussion.  That might be in addition to or instead of the information about the original post.  And the time part of date/time is very much desired.

To illustrate . . .

![#01](https://github.com/community/community/assets/75588964/f1b32adf-7d15-41fc-80c4-b157887f254c)

I've put boxes around the things I'm talking about.  What you see there is when the first post in the thread was dropped on here.  I'm looking for the date & time of the most recent post, including any reply, within the discussion.  I would be fine if that were in addition to what is there or instead of what is there.

**Comment by gil-son** [comment]
Try to do that:

![github02](https://github.com/community/community/assets/72712095/d2232c8e-4675-4c65-ad85-1bc2333ce14d)

is:open category:Accessibility sort:top created:>=2024-01-31T12:00:00

In this example, "T12:00:00" represents 12:00 PM. Adjust the time according to your specific requirements

**Comment by GrampaWildWilly** [comment]
Thank you for your response.

That's not what I'm looking for.  The result of the search still shows the date of the original post in a thread, not the most recent intervention.  And continually updating the search terms seems like a rather clumsy way of doing things.  I would have to be repeatedly changing the search terms.  I can already sort the list descending by most recent post.  I want the results to show the time stamp in the search results.  That way, I can just reload the page & see when a thread got an update.

**Comment by ghostinhershell** [comment]
Thanks for sharing and contributing to our community @GrampaWildWilly!

I’ll go ahead and file this with the applicable team so that it’s on their radar. In the meantime, you can check out our [public roadmap](https://github.com/github/roadmap/projects/1) to see the kinds of work we are focusing on for the coming year. Smaller fixes won’t be listed, but are often included when work is happening adjacent to it.

Thanks again!

**Comment by GrampaWildWilly** [comment]
Thank you so much!  I didn't know if this was the right place to make this suggestion.  I guess it was close enough.  I clicked the roadmap thing & was a bit puzzled when I saw it was dated 2021.  But I went ahead & clicked the link there & got current information.  My idea here would definitely fall under the small heading.  My notification that it's been implemented will be when I see it in discussion lists.  Thanks again for your response.
